data-transform.qmd

```diff
@@ -347,7 +347,7 @@ In this situation, the first challenge is often just focusing on the variables y
       select(!year:day)
     ```
 
-    You can also use `-` instead of `!` (and you're likely to see that in the wild); we recommend `!` because it reads as "not", and combines well with `&` and `|`.
+    Historically this operation was done with `-` instead of `!`, so you're likely to see that in the wild. These two operators serve the same purpose but with subtle differences in behaviour. We recommend to consistently use `!` because it reads as "not" and combines well with `&` and `|`.
```

Comment:
```suggestion
    Historically this operation was done with `-` instead of `!`, so you're likely to see that in the wild. These two operators serve the same purpose but with subtle differences in behavior. We recommend using `!` because it reads as "not" and combines well with `&` and `|`.
```